### ISSUE TITLE
Fatal Error: TermToConstTermUsingModel with multiple instances

### ISSUE THREAD

**Issue by ahorn** [stp/stp#97]
Consider the following minor change:

````diff
index c630461..4769adb 100644
--- a/examples/simple/example.c
+++ b/examples/simple/example.c
@@ -11,6 +11,7 @@ int main(int argc, char** argv)
 {
     int width=8;
     VC handle = vc_createValidityChecker();
+    VC handle2 = vc_createValidityChecker();

     // Create variable "x"
     Expr x = vc_varExpr(handle, "x", vc_bvType(handle, width));
````

Compile and run the modified file as follows:

    $ g++ -I/path/to/stp/target/include -L/path/to/stp/target/lib -lstp example.c
    $ ./a.out

The output is as follows:

````
######First Query
Assertions:
ASSERT( TRUE );
Query:
QUERY(TRUE );
Query is VALID


######Second Query
Assertions:
ASSERT( TRUE );
Fatal Error:  TermToConstTermUsingModel: termITE: cannot compute ITE conditional against model:

426:(ITE
  422:unconstrained_1
  14:0x02
  18:0x03)

0
````

This effectively prevents more than one instance of STP to exist. How could I work around this issue?

**Comment by TrevorHansen** [comment]
This is fixed in e5f0ddeae9faf1c7c27b9559bc7a4e1e5d1d39c0

**Comment by TrevorHansen** [comment]
oops. actually in 54d40c621cbadda291fd50edaccb4a5058534f72
